code/modules/shuttle/supply.dm

```diff
@@ -538,16 +538,29 @@
 /datum/economy/simple_seller/tech_levels/begin_sell(obj/docking_port/mobile/supply/S)
 	temp_tech_levels = SSeconomy.tech_levels.Copy()
 
+/datum/economy/simple_seller/tech_levels/proc/getPrice(tech_rarity, tech_level, sold_level = null)
+	// Calculates tech disk's supply points sell cost
+	if(!sold_level)
+		sold_level = 1
+
+	if(sold_level >= tech_level)
+		return 0
+
+	var/cost = 0
+	for(var/i = sold_level + 1, i <= tech_level, i++)
```

Comment:
```suggestion
	for(var/i in (sold_level + 1) to tech_level)
```
The syntax you're using for a `for` loop here is propery, however we use this format for the para codebase, it's a bit more readable.